content/id/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Poseidon-Firmament - Sebuah Penjadwal Alternatif
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+**Rilis saat ini dari Penjadwal Poseidon-Firmament adalah rilis <code> alfa </code>.**
```

Comment:
alpha doesn't have to be translated, since it's related to the Kubernetes versioning
```suggestion
**Rilis saat ini dari Penjadwal Poseidon-Firmament adalah rilis <code> alpha </code>.**
```